src/System.Runtime/tests/System/DateTimeOffsetTests.cs

```diff
@@ -357,6 +532,33 @@ public static void UtcDateTime()
             Assert.Equal(now.ToUniversalTime(), dateTimeOffset.UtcDateTime);
         }
 
+        [Fact]
+        public static void UtcNow()
+        {
+            Assert.NotSame(DateTimeOffset.UtcNow, DateTimeOffset.UtcNow);
```

Comment:
This could fail; it's possible that two calls to UtcNow happen close enough together that they return the same value.